packages/console/src/pages/EnterpriseSso/Guide/XmlFileUploader/index.module.scss

```diff
@@ -0,0 +1,67 @@
+@use '@/scss/underscore' as _;
+
+.fileInput {
+  display: none;
+}
+
+.preview {
+  display: flex;
+  max-width: 400px;
+  align-items: center;
+  flex-direction: row;
+  padding: _.unit(5) _.unit(5) _.unit(5) _.unit(4);
+  background: var(--color-layer-1);
+  color: var(--color-text);
+  border-color: var(--color-border);
+  border-width: 1px;
+  border-style: solid;
+  border-radius: _.unit(3);
+
+  .fileIcon {
+    > svg {
+      display: flex;
+      max-width: 100%;
+      max-height: 100%;
+    }
+  }
+
+  .fileInfo {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: _.unit(0.5);
+    flex: 1;
+    margin-left: _.unit(2);
+
+    .fileName {
+      font: var(--font-label-2);
+    }
+
+    .fileSize {
+      font: var(--font-label-3);
+      color: var(--color-text-secondary);
+    }
+  }
+
+  .delete {
+    display: flex;
+    width: 28px;
+    height: 28px;
+    border-radius: _.unit(1.5);
+    border: none;
+    padding: _.unit(1);
+    background-color: transparent;
+
+    .icon {
+      max-width: 100%;
+      max-height: 100%;
+      flex: 1;
```

Comment:
I think you can simply use `width: 100%`

Reply:
Removed redundant CSS